eland/operations.py

```diff
@@ -812,7 +915,7 @@ def _map_pd_aggs_to_es_aggs(pd_aggs):
         if extended_stats_calls >= 2:
             es_aggs = [
                 ("extended_stats", es_agg)
-                if es_agg in extended_stats_es_aggs
+                if not isinstance(es_agg, tuple) and (es_agg in extended_stats_es_aggs)
```

Comment:
I'm not sure we need this, the `extended_stats_es_aggs` doesn't have any tuples so this shouldn't be an issue right?

Reply:
So The following is happening
Because - `extended_stats_es_aggs = {"avg", "min", "max", "sum"}` is a set
```python
>>> ("percentiles", [50.0]) in extended_stats_es_aggs
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
```
The following will work:
```python
>>> extended_stats_es_aggs = ["avg", "min", "max", "sum"]
>>> ("percentiles", [50.0]) in extended_stats_es_aggs
False
>>> ("extended_stats", "std_deviation") in extended_stats_es_aggs
False
```
Should I change it to list? Because we are just comparing them..

Reply:
What are your thoughts on changing `(percentiles, [50.0])` instead to `(percentiles, (50.0,))`, then it'd be hashable and we don't have to change anything here?

Reply:
I just used a list, tuple works too. Let me change and check.